go/vt/vtgate/evalengine/fn_time.go

```diff
@@ -1375,7 +1375,7 @@ func dateTimeUnixTimestamp(env *ExpressionEnv, date eval) evalNumeric {
 			case *evalDecimal:
 				prec = d.length
 			case *evalBytes:
-				if d.isHexLiteral {
+				if d.isHexOrBitLiteral() {
 					return newEvalInt64(0)
```

Comment:
Should this return a `uint64` in case of a hex literal?

Reply:
Fixed